<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectableDetectDoubleFeedSettingsDialog</class>
 <widget class="QDialog" name="SelectableDetectDoubleFeedSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/icon.qrc">
    <normaloff>:/escan2_app.ico</normaloff>:/escan2_app.ico</iconset>
  </property>
  <widget class="DetectionAreaMaxDoubleSpinBox" name="Max_doubleSpinBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>77</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>297</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true">Close</string>
   </property>
  </widget>
  <widget class="DetectionAreaMinDoubleSpinBox" name="Min_doubleSpinBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>77</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
  </widget>
  <widget class="QStackedWidget" name="Det_Area_stackedWidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>77</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="mm_page">
    <widget class="QLabel" name="mm_label1">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>6</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>5</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">-</string>
     </property>
    </widget>
    <widget class="QLabel" name="mm_label2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>6</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">mm</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="inchi_page">
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="inchi_label2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>6</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">in.</string>
     </property>
    </widget>
    <widget class="QLabel" name="inchi_label1">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>6</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">in.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>5</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">-</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="PaperThicknessComboBox" name="Paper_Thickness_comboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>46</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="Detection_Area_label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true">CONFIG_DOC_016</string>
   </property>
  </widget>
  <widget class="QLabel" name="Paper_Thickness_label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>49</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true">CONFIG_DOC_012</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Ultrasonic_sensor_checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>17</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true">CONFIG_DOC_007</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Scanned_image_checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>107</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true">Scanned image</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Paper_Protection_checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>164</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true">Paper Protection</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>137</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Description_textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>197</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="html">
    <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="Paper_Thickness_colon_label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>49</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true">:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Detection_Area_colon_label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>80</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true">:</string>
   </property>
  </widget>
  <zorder>Close_pushButton</zorder>
  <zorder>Det_Area_stackedWidget</zorder>
  <zorder>Paper_Thickness_comboBox</zorder>
  <zorder>Detection_Area_label</zorder>
  <zorder>Paper_Thickness_label</zorder>
  <zorder>Ultrasonic_sensor_checkBox</zorder>
  <zorder>Scanned_image_checkBox</zorder>
  <zorder>Paper_Protection_checkBox</zorder>
  <zorder>line</zorder>
  <zorder>Description_textBrowser</zorder>
  <zorder>Min_doubleSpinBox</zorder>
  <zorder>Max_doubleSpinBox</zorder>
  <zorder>Paper_Thickness_colon_label</zorder>
  <zorder>Detection_Area_colon_label</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaperThicknessComboBox</class>
   <extends>QComboBox</extends>
   <header>paperthicknesscombobox.h</header>
  </customwidget>
  <customwidget>
   <class>DetectionAreaMinDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>detectionareamindoublespinbox.h</header>
   <slots>
    <slot>setMaxValue(double)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>DetectionAreaMaxDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>detectionareamaxdoublespinbox.h</header>
   <slots>
    <slot>setMinValue(double)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons/icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Close_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SelectableDetectDoubleFeedSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Max_doubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Min_doubleSpinBox</receiver>
   <slot>setMaxValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Min_doubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Max_doubleSpinBox</receiver>
   <slot>setMinValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Paper_Thickness_comboBox</sender>
   <signal>activated(int)</signal>
   <receiver>SelectableDetectDoubleFeedSettingsDialog</receiver>
   <slot>onValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Ultrasonic_sensor_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SelectableDetectDoubleFeedSettingsDialog</receiver>
   <slot>onValueChangedBool(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Scanned_image_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SelectableDetectDoubleFeedSettingsDialog</receiver>
   <slot>onValueChangedBool(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Paper_Protection_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SelectableDetectDoubleFeedSettingsDialog</receiver>
   <slot>onValueChangedBool(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onValueChangedBool(bool)</slot>
  <slot>onValueChanged(int)</slot>
 </slots>
</ui>
